pom.xml

```diff
@@ -38,12 +38,12 @@
     <dependency>
       <groupId>io.vertx</groupId>
       <artifactId>vertx-core</artifactId>
-      <version>3.5.4</version>
+      <version>${vertx.version}</version>
```

Comment:
Instead of introducing ${vertx.version} consider using Bill of Materials (Maven BOM): https://github.com/vert-x3/vertx-stack#bills-of-materials-maven-bom
This also sets the netty and jackson version: https://github.com/vert-x3/vertx-dependencies/blob/master/pom.xml#L50-L53